File: Forum.Domain/Theme.cs
using Forum.Domain.Identity;

namespace Forum.Domain
{
    public class Theme : BaseEntity
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime CreatingTime { get; set; }
        public int AuthorId { get; set; }
        public User Author { get; set; }
        public IList<Message> Messages { get; set; }
        public Section Section { get; set; }
        public int SectionId { get; set; }
    }
}

File: Forum.Angular/src/app/_models/theme/theme.ts
import { Message } from "../message/message"
import { User } from "../user/user"

export interface Theme {
    id: number;
    title: string;
    description: string;
    creatingTime: Date;
    authorId: number;
    sectionId: number;
    author?: User;
    messages?: Message[];
}